CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Change "cross service header" spec filename ([PR #3743](https://github.com/alphagov/govuk_publishing_components/pull/3743))
```

Comment:
I don't think this warrants a changelog entry - it's more of a maintenance change, and it's not relevant to applications since the tests are confined to the gem.

Reply:
Yep see what you mean. I'll remove the entry. Thanks for the approval.